nautobot/ipam/tests/test_api.py

```diff
@@ -114,55 +113,31 @@ def get_deletable_object_pks(self):
         return [rir.pk for rir in RIRs]
 
 
-class AggregateTest(APIViewTestCases.APIViewTestCase):
-    model = Aggregate
-    brief_fields = ["display", "family", "id", "prefix", "url"]
-    bulk_update_data = {
-        "description": "New description",
-    }
-
-    @classmethod
-    def setUpTestData(cls):
-
-        rir = RIR.objects.filter(is_private=False).first()
-
-        cls.create_data = [
-            {
-                "prefix": "12.0.0.0/8",
-                "rir": rir.pk,
-            },
-            {
-                "prefix": "2d00::/8",
-                "rir": rir.pk,
-            },
-            {
-                "prefix": "17.0.0.0/16",
-                "rir": rir.pk,
-            },
-        ]
-
-
 class PrefixTest(APIViewTestCases.APIViewTestCase):
     model = Prefix
     brief_fields = ["display", "family", "id", "prefix", "url"]
     choices_fields = []
 
     @classmethod
     def setUpTestData(cls):
+        rir = RIR.objects.filter(is_private=False).first()
         cls.statuses = Status.objects.get_for_model(Prefix)
         cls.status_active = cls.statuses.get(slug="active")
         cls.create_data = [
             {
                 "prefix": "192.168.4.0/24",
                 "status": cls.status_active.pk,
+                "rir": rir.pk,
             },
             {
                 "prefix": "2001:db8:abcd:12::/80",
                 "status": cls.status_active.pk,
+                "rir": rir.pk,
             },
             {
                 "prefix": "192.168.6.0/24",
                 "status": cls.status_active.pk,
+                "rir": rir.pk,
```

Comment:
Let's have one of these with no `rir` (to validate that it's an optional field) and at least one that specifies a `date_allocated`?

Reply:
Probably also should add some `type` values in here as well...